lib/modules/datasource/galaxy-collection/index.spec.ts

```diff
@@ -163,5 +163,47 @@ describe('modules/datasource/galaxy-collection/index', () => {
       expect(res).toBeDefined();
       expect(res?.releases).toHaveLength(3);
     });
+
+    it('returns null but matches automation hub url', async () => {
```

Comment:
```suggestion
    it('returns null but matches automation hub URL', async () => {
```

We should capitalize abbreviations like URL.